src/components/conversations/ConversationView.tsx

```diff
@@ -38,6 +34,7 @@ interface SingleConvoProps {
   participants: ReduxParticipant[];
   messages: ReduxMessage[];
   typingInfo: string[];
+  timeFormat: boolean;
```

Comment:
`timeFormat` is logically a string; if this field is boolean it should be named something like `use24hTimeFormat` perhaps?